include/openenclave/internal/crypto/init.h

```diff
@@ -15,5 +15,6 @@ void oe_crypto_initialize(void);
 #define OE_SYMCRYPT_ENGINE_INVALID 3
 
 int oe_is_symcrypt_engine_available();
+int oe_is_symcrypt_provider_available();
```

Comment:
Does this do anything?

Reply:
This function and its relative above are used in the tests "symcrypt_enigne" and "symcrypt_provider". I have added "symcrypt_provider' and fixed "symcrypt_engine"